### PR TITLE
Activate stylelint extension when opening html, vue and more types.

### PR DESCRIPTION
**Pull request by trainiac**
Activate plugin when opening html or vue files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
     "onLanguage:postcss",
     "onLanguage:scss",
     "onLanguage:sugarss",
+    "onLanguage:vue",
```
Comment (shinnn) [inline package.json]: Add more. https://github.com/shinnn/stylelint-vscode/issues/9#issuecomment-347111177

**Comment by shinnn** [comment]
> Update package.json

Write more appropriate pull request title.

**Comment by trainiac** [comment]
@shinnn Addressed your requested changes. Please review. Thanks

File: package.json [hunk 2]
```diff
@@ -29,10 +29,16 @@
...
+    "onLanguage:html",
     "onLanguage:less",
+    "onLanguage:markdown",
     "onLanguage:postcss",
     "onLanguage:scss",
     "onLanguage:sugarss",
+    "onLanguage:svg",
+    "onLanguage:vue",
+    "onLanguage:vue-html",
+    "onLanguage:xml",
```
Comment (shinnn) [inline package.json]: @trainiac Do you know any document IDs else vscode-stylelint should support? I took a glance to the VSCode built-in document IDs and the user-land ones, and I couldn't find anymore.

**Comment by trainiac** [comment]
@shinnn Closing and reopening with the stuff you want.

**Comment by shinnn** [comment]
No need to close this. You can update this branch.

File: package.json (hunk 2 above)
Comment (trainiac) [reply package.json]: maybe stylus?

Comment (trainiac) [reply package.json]: Now seeing your comments about supporting upstream sytanxes, so stylus is not needed.  I think this list is good, you can always add more.

**Comment by trainiac** [comment]
@shinnn let me know if there is anything else I can do.

**Comment by trainiac** [comment]
sorry for all the trouble on such a simple request 😂

**Comment by trainiac** [comment]
pug files could be another one

**Comment by shinnn** [comment]
Can stylelint check `.pug` files without using any external plugins? [As far as I can see](https://github.com/stylelint/stylelint/blob/78c0a5406a88e5f1630ef261c6af976bd1977802/docs/user-guide/css-processors.md#parsing-non-standard-syntax), it cannot.

**Comment by trainiac** [comment]
You are correct, indeed. There would need to be processor for them.

**Comment by trainiac** [comment]
@shinnn I need to keep making changes to my master branch so reopening request with a feature branch

**Comment by shinnn** [comment]
OK, I don't merge your pull request. I don't want to see your messy duplicate patches anymore. I'll just push it to `master` by myself.

**Comment by trainiac** [comment]
@shinnn Aren't you a charmer.  Any chance you can update master with this and make a new release?

File: package.json (hunk 2 above)
Comment (gucong3000) [reply package.json]: `stylelint` does not support stylus.

**Comment by shinnn** [comment]
https://github.com/shinnn/vscode-stylelint/commit/c554d7afa6e2f3af7bb4dd5b09d14206ced72883

**Pull request closed without merging**